src/api/integrations/event/webhook/webhook.controller.ts

```
@@ -6,6 +6,8 @@
 import { Logger } from '@config/logger.config';
 import { BadRequestException } from '@exceptions';
 import axios, { AxiosInstance } from 'axios';
+import { isURL } from 'class-validator';
```

ESLint

[error] 'isURL' is defined but never used.

(@typescript-eslint/no-unused-vars)